MahApps.Metro/Styles/Fonts.xaml

```diff
@@ -25,9 +25,6 @@
 
     <System:Double x:Key="StatusBarFontSize">12</System:Double>
 
-    <System:Double x:Key="DialogTitleFontSize">26</System:Double>
-    <System:Double x:Key="DialogMessageFontSize">15</System:Double>
-
     <System:Double x:Key="FlyoutHeaderFontSize">20</System:Double>
```

Comment:
Can you try reuse this existing font sizes? Otherwise it's a breaking change and I can't add this PR for 1.3.0